components.d.ts

```diff
@@ -58,7 +58,6 @@ declare module '@vue/runtime-core' {
     DiscordIcon: typeof import('./src/components/_global/icons/brands/DiscordIcon.vue')['default']
     EmailIcon: typeof import('./src/components/_global/icons/EmailIcon.vue')['default']
     GithubIcon: typeof import('./src/components/_global/icons/brands/GithubIcon.vue')['default']
-    Jazzicon: typeof import('./src/components/_global/JazzIcon/Jazzicon.vue')['default']
```

Comment:
Why was this removed?

Reply:
I investigated a little bit and couldn't find the reason. It is not affecting the DX so I will review it again in case it does.